tools/docker/Dockerfile.jinja

```diff
@@ -138,10 +165,10 @@ FROM build-ovmf as ovmf
 WORKDIR /root
 RUN git clone --depth 1 --branch edk2-stable202402 --recurse-submodules --shallow-submodules https://github.com/tianocore/edk2.git
 WORKDIR /root/edk2
-RUN source ./edksetup.sh \ 
+RUN /bin/bash -c "source ./edksetup.sh \ 
```

Comment:
What is the problem with the original form, which does not use `/bin/bash -c`?

Reply:
When I build a docker image locally, I found that the shell cannot find ` source` without `/bin/bash -c`. I am not sure whether this situation will occur in the GitHub CI.

Reply:
ok